plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -142,7 +147,8 @@ function ProductCard( props: ProductCardProps ): JSX.Element {
 			rel="noopener noreferrer"
 			onClick={ () => {
 				recordTracksEvent( 'marketplace_product_card_clicked', {
-					product: product.title,
+					product: product.id,
```

Comment:
I would just suggest we change the Tracks key to make it clear what we mean by "product". [Orlando](p1725895324171879/1724801227.652689-slack-C021P7KR98T) has no objection.

```suggestion
					product_id: product.id,
```